doc/source/data/batch_inference.rst

```diff
@@ -1,823 +1,455 @@
 .. _batch_inference_home:
 
-Running Batch Inference
-=======================
+End-to-end: Offline Batch Inference
+===================================
 
-.. note::
-
-    In this tutorial you'll learn what batch inference is, why you might want to use
-    Ray for it, and how to use Ray Data effectively for this task.
-    If you are familiar with the basics of inference tasks, jump straight to
-    code in the :ref:`quickstart section <batch_inference_quickstart>`, our detailed
-    :ref:`walk-through<batch_inference_walk_through>`,
-    or our :ref:`in-depth guide for PyTorch models<batch_inference_advanced_pytorch_example>`.
+.. tip::
 
-Batch inference refers to generating model predictions on a set of input data.
-The model can range from a simple Python function to a complex neural network.
```

Comment:
all this stuff is info I deeply believe new users want to read.

Reply:
This is still there in the first paragraph right?